hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -374,16 +394,17 @@ public ContainerBackgroundTaskResult deleteViaSchema2(
             deleteTxns = dnStoreTwoImpl.getDeleteTransactionTable();
         List<DeletedBlocksTransaction> delBlocks = new ArrayList<>();
         int totalBlocks = 0;
+        int blockLimit = 0;
```

Comment:
Lets rename this variable to numBlocks or sth similar.

Reply:
Renamed to numBlocks